website/source/api/acl/acl.html.md

```diff
@@ -263,3 +263,112 @@ agent_prefix "" {
    policy = "read"
 }
 ```
+
+## Login to Auth Method
+
+This endpoint was added in Consul 1.5.0 and is used to exchange an [auth
+method](/docs/acl/acl-auth-methods.html) bearer tokens for a newly-created
+Consul ACL token.
+
+| Method | Path                         | Produces                   |
+| ------ | ---------------------------- | -------------------------- |
+| `POST` | `/acl/login`                 | `application/json`         |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/index.html#blocking-queries),
+[consistency modes](/api/index.html#consistency-modes),
+[agent caching](/api/index.html#agent-caching), and
+[required ACLs](/api/index.html#acls).
+
+| Blocking Queries | Consistency Modes | Agent Caching | ACL Required |
+| ---------------- | ----------------- | ------------- | ------------ |
+| `NO`             | `none`            | `none`        | `none`       |
+
+### Parameters
+
+- `AuthMethod` `(string: <required>)` - The name of the auth method to use for login.
+
+- `BearerToken` `(string: <required>)` - The bearer token to present to the
+  auth method during login for authentication purposes. For the Kubernetes Auth
```

Comment:
By capitalizing "Kubernetes Auth Method" we are putting emphasis here, is that intentional? Typically, only products should be capitalized.

Reply:
fixed